#! /bin/sh /usr/share/dpatch/dpatch-run
## refocus_autoconf.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: cleaning up autoconf weirdness.

@DPATCH@
diff -urNad gimp-plugin-registry~/refocus/src/configure.in gimp-plugin-registry/refocus/src/configure.in
--- gimp-plugin-registry~/refocus/src/configure.in	2003-01-30 22:29:29.000000000 +0100
+++ gimp-plugin-registry/refocus/src/configure.in	2007-10-15 00:56:21.000000000 +0200
@@ -15,30 +15,6 @@
 AC_STDC_HEADERS
 AC_PROG_RANLIB
 
-# Check if the user has ATLAS installed in ./lib-atlas
-fw_save_LIBS=$LIBS
-LIBS=-L./lib-atlas/lib ${LDFLAGS}
-AC_CHECK_LIB(lapack, clapack_dgesv, 
-AC_MSG_RESULT([using atlas in lib-atlas/lib])
-AC_DEFINE(HAVE_ATLAS)
-have_atlas=yes
-LAPACK_LIB_DIR='${top_srcdir}/lib-atlas/lib'
-LAPACK_INCLUDE_DIR='${top_srcdir}/lib-atlas/include'
-,
-AC_MSG_RESULT([using unoptimized lapack in lib])
-have_atlas=no
-LAPACK_LIB_DIR='${top_srcdir}/lib'
-LAPACK_INCLUDE_DIR='${top_srcdir}/lib'
-,
-[-lcblas -latlas])
-LIBS=$fw_save_LIBS
-AC_SUBST(LAPACK_LIB_DIR)
-AC_SUBST(LAPACK_INCLUDE_DIR)
-AM_CONDITIONAL(HAVE_ATLAS, test x${have_atlas} = xyes)
-
-AM_PATH_GIMP(1.2.0)
-AM_PATH_GTK_2_0(2.0.0)
-AM_PATH_GLIB_2_0(2.0.0)
 
 # This is a check for gtk-doc which you can insert into your configure.in.
 # You shouldn't need to change it at all.
@@ -96,15 +72,9 @@
    AC_DEFINE(PREVIEW_DEBUG)
 fi
 
-# If we have gcc set the CFLAGS
-# This is done here because otherwise configure would use
-# these flags for compiling test-programs.
-if test "$GCC" = yes; then
-   CFLAGS="-Wall -ansi -pedantic -ggdb -fomit-frame-pointer -O3 -funroll-all-loops"
-fi
 
 #Check if erf is defined in the mathlibrary
 AC_CHECK_LIB(m, erf, AC_DEFINE(HAVE_ERF))
 
-AC_OUTPUT([Makefile src/Makefile lib/Makefile doc/Makefile gtk-doc/Makefile ])
+AC_OUTPUT([Makefile src/Makefile doc/Makefile gtk-doc/Makefile ])
 
